### ISSUE TITLE
Create new Phenodigm evidence table in rewrite version

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1242]
On the evidence page in the rewrite, can we please create a new data table that displays the Phenodigm evidence data?

**Example GraphQL API query**
[FGF8 and Joubert syndrome with orofaciodigital defect](https://api-beta-dot-open-targets-eu-dev.appspot.com/api/v4/graphql/browser?query=query%20assocs%20%7B%0A%20%20disease(efoId%3A%20%22Orphanet_2754%22)%20%7B%0A%20%20%20%20name%0A%20%20%20%20evidences(ensemblIds%3A%20%5B%22ENSG00000107831%22%5D%2C%20enableIndirect%3A%20true%2C%20datasourceIds%3A%20%5B%22phenodigm%22%5D)%20%7B%0A%20%20%20%20%20%20count%0A%20%20%20%20%20%20rows%7B%0A%20%20%20%20%20%20%20%20disease%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20target%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20approvedSymbol%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20diseaseFromSource%0A%20%20%20%20%20%20%20%20biologicalModelGeneticBackground%0A%20%20%20%20%20%20%20%20biologicalModelAllelicComposition%0A%20%20%20%20%20%20%20%20diseaseModelAssociatedModelPhenotypes%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20label%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20diseaseModelAssociatedHumanPhenotypes%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20label%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20score%0A%20%20%20%20%20%20%20%20resourceScore%0A%20%20%20%20%20%20%20%20resourceScoreType%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D&operationName=assocs)

**Data table**

| Data table column | Disease/phenotype | Reported disease/phenotype | Human phenotypes | Mouse phenotypes | Mouse model allelic composition |
| --- | --- | --- | --- | --- | --- |
| **GraphQL API field** | `evidences.rows[x].disease.name` | `evidences.rows[x].diseaseFromSource` | `evidences.rows[x].diseaseModelAssociatedHumanPhenotypes[y].label` | `evidences.rows[x].diseaseModelAssociatedModelPhenotypes[y].label` | `evidences.rows[x].biologicalModelAllelicComposition` & `evidences.rows[x].biologicalModelGeneticBackground`
| **Sample data** | Joubert syndrome with orofaciodigital defect | Joubert Syndrome 22 | <ul><li>Postaxial polydactyly</li><li>Cerebellar vermis hypoplasia</li><li>Molar tooth sign on MRI</li></ul> | <ul><li>absent cerebellum</li><li>abnormal craniofacial development</li><li>abnormal pituitary gland morphology</li><li>decreased embryo size</li><li>prenatal lethality, incomplete penetrance</li></ul>  | Fgf8<tm1.1Mrt>/Fgf8<tm1.2Mrt> involves: 129P2/OlaHsd

Within the data table, can we please do the following?

1. Link the value in the "Disease/phenotype" column to the relevant profile page in the Platform
2. Display the values in the "Human phenotypes" and "Mouse phenotypes" columns as an unordered list
3. Link the values in the "Mouse phenotypes" columns to MGI using identifiers.org, using the ID found in the `evidences.rows[x].diseaseModelAssociatedModelPhenotypes[y].id` field (e.g. [absent cerebellum](http://www.informatics.jax.org/vocab/mp_ontology/MP:0000850))
4. Format the value in the "Mouse model allelic composition" column to match how we display mouse model data on the target profile page (e.g. [ESR1](https://alpha.targetvalidation.org/target/ENSG00000091831))

**Comment by andrewhercules** [comment]
Can we please make some small adjustments to the data table?

**Table functionality**
- [ ] Similar to what has been proposed with the Europe PMC evidence data table - see #1248 - please default to showing only 5 rows per page as the detail view can become quite long with 10+ entries (e.g. [FGF8 and genetic disorder](https://alpha.targetvalidation.org/evidence/ENSG00000107831/EFO_0000508))
- [ ] Allow user to search by the following columns = "Disease/phenotype", "Reported disease/phenotype", "Human phenotypes", and "Mouse phenotypes"

**External links**
- [ ] Remove external link to Open Targets Genetics Portal for content in the "Reported disease/phenotype" column as the links do not work

**Comment by andrewhercules** [comment]
With the new iteration of the GraphQL API, can we please make a small adjustment to the PhenoDigm evidence data table?

Sample query: [FGF8 and Joubert syndrome with orofaciodigital defect](https://api-beta-dot-open-targets-eu-dev.appspot.com/api/v4/graphql/browser?query=query%20evidenceData%20%7B%0A%20%20disease(efoId%3A%20%22Orphanet_2754%22)%20%7B%0A%20%20%20%20name%0A%20%20%20%20evidences(ensemblIds%3A%20%5B%22ENSG00000107831%22%5D%2C%20enableIndirect%3A%20true%2C%20datasourceIds%3A%20%5B%22phenodigm%22%5D)%20%7B%0A%20%20%20%20%20%20count%0A%20%20%20%20%20%20rows%7B%0A%20%20%20%20%20%20%20%20disease%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20target%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20approvedSymbol%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20diseaseFromSource%0A%20%20%20%20%20%20%20%20biologicalModelGeneticBackground%0A%20%20%20%20%20%20%20%20biologicalModelAllelicComposition%0A%20%20%20%20%20%20%20%20diseaseModelAssociatedModelPhenotypes%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20label%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20diseaseModelAssociatedHumanPhenotypes%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20label%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20targetInModel%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D&operationName=evidenceData) 

- [ ] After the "Mouse phenotypes" column, add a column called "Mouse gene" and display the `evidences.rows[x].targetInModel` value and link to Ensembl.org using identifiers.org - `https://identifiers.org/ensembl/` + `evidences.rows[x].targetInModel` (e.g. [ENSMUSG00000025219](https://identifiers.org/ensembl/ENSMUSG00000025219))

**Comment by d0choa** [comment]
- [ ] Add reported disease/phenotype as tooltip in disease column. The tooltip content would be “Reported disease or phenotype:\n{diseaseFromSource}”
- [ ] For mouse phenotypes and human phenotypes use the drawer and display “n phenotypes”. Also display “1 phenotype” for consistency.
- [ ] Remove the mouse gene (again). Not really informative. Sorry about this.
- [ ] We might be ok coming back to showing 10 rows by default
